README.md

```diff
@@ -47,14 +47,18 @@ Aero. If you are using windows, its highly recommended to use WSL 2.
 
 Before building Aero, you need the following things installed:
 - `rust` (should be the **latest nightly**)
-- `qemu` (optional: required if you want to run it in the Qemu emulator)
 - `nasm`
+- `qemu` (optional: required if you want to run it in the Qemu emulator)
 
-If you are building Aero with sysroot then the following dependencies are additionally required:
+If you are building Aero with sysroot, run the following helper script to install additional dependencies.
 ```sh
-# Arch Linux:
-sudo pacman -S bash coreutils make patch tar gzip binutils gcc git subversion mercurial curl wget xz nasm mtools meson perl m4 texinfo groff gettext expat bison flex help2man openssl gperf rsync libxslt python python-pip python-mako xcb-proto xorriso boost-libs cmake
+./tools/deps.sh
```

Comment:
The file is actually present in the `deps` directory. So the path would be `./tools/deps/deps.sh` or just move the file to the `tools` for simplicity.

Reply:
Alright, I went ahead and fixed this.